.github/workflows/auto-merge.yml

```
@@ -1,36 +1,17 @@
-          
-          if [ "$reviews" -gt 0 ]; then
-            echo "PR #$pr_number is approved, enabling auto-merge"
-            
-            # Enable auto-merge with squash strategy
-            gh pr merge $pr_number --auto --squash --delete-branch
-            
-            echo "Auto-merge enabled for dependabot PR #$pr_number"
-          else
-            echo "PR #$pr_number is not yet approved, skipping auto-merge"
-          fi
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)